products/error_tracking/frontend/components/ExceptionCard/Tabs/StacktraceTab.tsx

```diff
@@ -96,8 +96,10 @@ function StacktraceIssueDisplay({
         ...stacktraceDisplayProps,
         renderLoading: (renderHeader: (props: ExceptionHeaderProps) => JSX.Element) =>
             renderHeader({
-                type: issue?.name ?? undefined,
-                value: issue?.description ?? undefined,
+                exception: {
+                    type: issue?.name ?? undefined,
+                    value: issue?.description ?? undefined,
+                } as any,
```

Comment:
just not sure what to do here since i dont have the exception yet
i could pass the 1st item from exceptionList if any to StacktraceIssueDisplay and then reuse here but is that the right thing to do?

Reply:
the problem is that we need them separated and concatenated in some components
@daibhin would you like to give a stab at figuring this out? Feels like this is more your domain than mine
i am happy to merge as it is but i dont disagree with your comment above

Reply:
This was more work than I expected. I think the problem here is that we were trying to show a title where it didn't make sense (use issue as a title when it should be an exception card). I've removed the reference to issues and just made a loading state instead